openslides/users/models.py

```diff
@@ -9,13 +11,17 @@
     Permission,
     PermissionsMixin,
 )
+from django.core import mail
 from django.db import models
 from django.db.models import Prefetch, Q
+from django.utils import timezone
 from jsonfield import JSONField
 
+from ..core.config import config
 from ..core.models import Projector
 from ..utils.collection import CollectionElement
 from ..utils.models import RESTModelMixin
+from ..utils.rest_api import ValidationError
```

Comment:
Ah this is wrong. Please see motion models and motion views and look for ValidationError and DjangoValidationError. In models we use Django's built in ValidationError. In the view you may have to catch it and raise DjangoRESTFramework stuff.

Reply:
Done